### PR TITLE
Extended onderwijsstructuur and added version triples

### PR DESCRIPTION
**Pull request by mvanbrab**
In a nutshell:
- skos;prefLabels are disambiguated
- short labels were added using skosxl
Full change list available in CHANGELOG.md

### CODE REVIEW

**Comment by mvanbrab** [comment]
I'll merge, fix the release date in CHANGELOG.md and tag after reviewers consent.

**Comment by mvanbrab** [comment]
I'll merge, fix the release date in CHANGELOG.md and tag after reviewers consent.

**Comment by mvanbrab** [comment]
Oops, see concepts missing in the skos:OrderedCollection's. Updating...

**Comment by mvanbrab** [comment]
I'm done with early-in-the-morning-thinking-about-what-i-forgot-yesterday changes. Reviewers, go ahead!

File: ilearn/generate-onderwijsstructuur/input/onderwijsstructuur.addendum.ttl [hunk 1]
```diff
@@ -0,0 +1,6 @@
+
```
Comment (bjdmeest) [inline ilearn/generate-onderwijsstructuur/input/onderwijsstructuur.addendum.ttl:4]: I would make this valid Turtle, normally concatenating turtle strings doesn't give issues, but then this file is valid and self-standing

File: ilearn/generate-onderwijsstructuur/tools/README.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+```shell
+python3 -m venv virtualenv
+source virtualenv/bin/activate
+python -V
+python -m pip install -r requirements.txt 
+deactivate
+```
+
+### 1.3 - YARRRML
+Download the right versie of the YARRRML parser (https://github.com/RMLio/yarrrml-parser.git) in this directory.
```
Comment (bjdmeest) [inline ilearn/generate-onderwijsstructuur/tools/README.md]: 'version'

File: ilearn/generate-onderwijsstructuur/tools/README.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+We use v4.15.0, so download https://github.com/RMLio/rmlmapper-java/releases/download/v4.15.0/rmlmapper-4.15.0-r361-all.jar.
+
+## 2 - Activation
+Execute this before running any next script. It activates the Python virtual environment:
+```shell
+source virtualenv/bin/activate
+```
+
+## 3 - Make the output
+Save all input files in `../input`, export the .xlsx file to a .csv and execute:
```
Comment (bjdmeest) [inline ilearn/generate-onderwijsstructuur/tools/README.md]: I would add a 'make sure to export to UTF-8' remark here

File: ilearn/onderwijsstructuur-extension/schemes/onderwijsstructuur.ttl [hunk 4]
```diff
@@ -0,0 +1,634 @@
+@prefix ocol: <https://w3id.org/onderwijs-vlaanderen/id/collectie/> .
```
Comment (bjdmeest) [inline ilearn/onderwijsstructuur-extension/schemes/onderwijsstructuur.ttl]: Is this file still relevant? can't it be removed?

File: ilearn/generate-onderwijsstructuur/input/onderwijsstructuur.addendum.ttl (hunk 1 above)
Comment (mvanbrab) [reply ilearn/generate-onderwijsstructuur/input/onderwijsstructuur.addendum.ttl:4]: Done, see https://github.com/i-Learn-SKOS/common-conceptschemes/pull/7/commits/a7abf3c19d83c2643d35405b10b761aef109e58f

File: ilearn/generate-onderwijsstructuur/tools/README.md (hunk 3 above)
Comment (mvanbrab) [reply ilearn/generate-onderwijsstructuur/tools/README.md]: Done, see https://github.com/i-Learn-SKOS/common-conceptschemes/pull/7/commits/516f02704c697a64f87fc9c501907a9d4d25073a

File: ilearn/onderwijsstructuur-extension/schemes/onderwijsstructuur.ttl (hunk 4 above)
Comment (mvanbrab) [reply ilearn/onderwijsstructuur-extension/schemes/onderwijsstructuur.ttl]: Done, see https://github.com/i-Learn-SKOS/common-conceptschemes/pull/7/commits/ccd84267e436482ff3050202e9df7e8056656e38

File: ilearn/generate-onderwijsstructuur/tools/README.md (hunk 2 above)
Comment (mvanbrab) [reply ilearn/generate-onderwijsstructuur/tools/README.md]: Done, see https://github.com/i-Learn-SKOS/common-conceptschemes/pull/7/commits/eaeeb7d5dd2cbf3ef48375cb3cd06be14c7092d8

**Comment by mielvds** [comment]
looks good to me!

**Pull request merged**